devel/CUPS/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                APPLE-CUPS
```

Comment:
let's not do all capitals perhaps, and just name the port as upstream does (i.e., using the `github` PG this will be set automatically and just become `cups`)